apps/server/.eslintrc.cjs

```diff
@@ -10,6 +10,7 @@ module.exports = {
     project: './tsconfig.json',
   },
   ignorePatterns: [
+    '.eslintrc.cjs',
     '.eslintrc.js',
     'jest.config.js',
     'jestSetup.ts',
```

Comment:
We should align on one of these files. (`.eslintrc.cjs` and `.eslintrc.js`--we don't need both). I created a ticket for cleaning up the linting in this project.